enhancements/insights/rapid-recommendations.md

```diff
@@ -0,0 +1,548 @@
+4. The pull request is reviewed (and either approved or denied) 
+   by a member of the Observability Intelligence (former CCX) team.
+5. When the pull request is merged, the new remote configuration is deployed 
+   to the remote configuration service on console.redhat.com.
+6. Insights Operators in connected clusters download the new remote configuration 
+   and start collecting the new data.
+7. The data requester makes use of the new data in newly incoming Insights Operator archives.
+
+### API Extensions
+
+This proposal does not add any API extension.
+
+### Topology Considerations
+
+#### Hypershift / Hosted Control Planes
+
+This proposal does not require any specific details to account for the HyperShift. 
+For now, we can simply extend the remote configuration file to 
+request gathering of specific HyperShift resources.
+
+#### Standalone Clusters
+
+This proposal is relevant to standalone (and especially connected) clusters. 
+Standalone cluster retrieves the remote configuration (based on its OCP version) and collects data accordingly. 
+
+#### Single-node Deployments or MicroShift
+
+This proposal may lead to increased resource consumption by the Insights Operator 
+(described in the [Excessive use of cluster resources](#excessive-use-of-cluster-resources) section), 
+but the operator can be disabled in these environments or in case of problems. 
+
+### Implementation Details/Notes/Constraints 
+
+#### Remote Configuration Service
+
+The Insights Operator configuration will replace the current [conditional gathering endpoint](../insights/conditional-data-gathering.md)
+
+```bash
+https://console.redhat.com/api/gathering/gathering_rules.json
+```
+
+with a new one:
+
+```bash
+https://console.redhat.com/api/gathering/v2/%s/gathering_rules.json
+```
+
+The `%s` parameter is the OCP version of the Insights Operator as set in the `.status.versions` map 
+of the Insights `clusteoperator.config.openshift.io` resource (consider partially upgraded clusters).
+The remote configuration service will be responsible for serving configuration that can be 
+successfully processed by an Insights Operator of the given version.
+
+The new endpoint will return both the current conditional gathering configuration and the new remote
+configuration for container logs gathering. 
+In future versions, the endpoint may also include remote configuration for node logs and API resources.
+
+#### Remote Configuration Processing
+
+The Insights Operator will try to download the remote configuration before creating each archive (every 2 hours).
+This enables fast recovery time in case a remote configuration change caused issues and had to be reverted.
+The frequency is the same as with the 
+[conditional gathering configuration](../insights/conditional-data-gathering.md) (since OCP 4.10). 
+40k connected clusters requesting a remote configuration update every two hours mean 11 requests per second on average.
```

Comment:
do we use something like cronjobs (which would synchronize every 2hrs) or somehow do it based on something that will smear over time?

Reply:
No, the Insights Operator does not use cronjobs. The interval is basically [measured from the start of the Insights Operator process](https://github.com/openshift/insights-operator/blob/master/pkg/controller/periodic/periodic.go#L237).

The previous sentence about conditional gathering was meant to say that the same system is already in use and is not causing any issues (none that we know of anyway). We wanted to quantify the traffic explicitly because this proposal will increase the remote configuration service complexity. There are no inherent traffic spikes and this proposal doesn't add any.

Reply:
ok